python/tvm/relay/frontend/tflite.py

```diff
@@ -247,6 +249,55 @@ def convert_softmax(self, op):
 
         return out
 
+    def convert_concatenation(self, op):
+        """ convert TFLite concatenation"""
+        try:
+            from tflite.Operator import Operator
+            from tflite.ConcatenationOptions import ConcatenationOptions
+            from tflite.BuiltinOptions import BuiltinOptions
+            from tflite.ActivationFunctionType import ActivationFunctionType
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        assert isinstance(op, Operator)
+        input_tensors = self.get_input_tensors(op)
+        output_tensors = self.get_output_tensors(op)
+        assert len(input_tensors) >= 1, "input tensors should greater than 1"
+        assert len(output_tensors) == 1, "output tensors should be 1"
+        in_exprs = [self.get_expr(input_tensor.tensor_idx) for input_tensor in input_tensors]
+        assert op.BuiltinOptionsType() == BuiltinOptions.ConcatenationOptions
```

Comment:
Suggest adding additional one blank line to separate input / output logic and Options like we did in other implementations.